core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -2673,3 +2673,25 @@ secret_configs:
   - cluster: app.ci
     name: registry-push-credentials-image-mirror-from-apici-to-appci
     namespace: ci
+- from:
```

Comment:
This is suprising in a "Doc how to provision ..." PR, please retitle to something less benign-sounding :D

Reply:
Removed the changes on secret-bootstrapper's config.
Let decide what to do during the implementation.
Here is only for demo.